packages/webapp/src/components/Modals/QuickAssignModal/index.jsx

```diff
@@ -103,21 +266,30 @@ export default function TaskQuickAssignModal({
       }
       icon={<Person />}
     >
-      <ReactSelect
-        data-cy="quickAssign-assignee"
-        defaultValue={selectedWorker}
-        label={t('ADD_TASK.ASSIGNEE')}
-        options={options}
-        onChange={setWorker}
-        style={{ marginBottom: '24px' }}
-        isSearchable
+      <Controller
+        control={control}
+        name={ASSIGNEE}
+        render={({ field: { onChange } }) => (
+          <ReactSelect
+            data-cy="quickAssign-assignee"
+            defaultValue={selectedWorker}
+            label={t('ADD_TASK.ASSIGNEE')}
+            options={options}
+            onChange={onChange}
+            style={{ marginBottom: 10 }}
+            isSearchable
+          />
+        )}
       />
+      {renderHourlyWageSection()}
```

Comment:
Please tell me if this is a bad idea or you do not want to do it. But I am thinking that this could get reused in the create task workflow since this same problem likely happens so it could maybe be its own separate component?

Reply:
I'll extract this and create a reusable component in my next ticket.